api/src/test/java/org/openmrs/PersonNameTest.java

```diff
@@ -383,7 +437,92 @@ public void getFullName_shouldReturnShortIfPersonNameFormatIsNull() throws Excep
 		pn.setFamilyNameSuffix("jr.");
 		pn.setDegree("3");
 		PersonName.setFormat("");
-		Assert.assertEquals(pn.getFullName(), "Sr. Taylor Bob Mark");
+
+		Assert.assertEquals("Sr. Taylor Bob Mark", pn.getFullName());
+	}
+
+	/**
+	 * @see PersonName(String, String, String)
+	 */
+	@Test
+	@Verifies(value = "should trim leading and trailing whitespaces on each name attribute", method = "PersonName")
+	public void personName_shouldTrimLeadingAndTrailingWhitespacesOnEachNameAttribute() throws Exception {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method names should not contain underscores](https://www.codacy.com/app/openmrs/openmrs-core/file/2318747982/issues/source?bid=3204099&fileBranchId=3204128#l449)